cluster-components/kuberos/ingress.yml

```diff
@@ -1,15 +0,0 @@
-apiVersion: extensions/v1beta1
```

Comment:
It should be safe to remove all the files under `cluster-components/kuberos/`

Reply:
ok

Reply:
fixed